clients/admin-ui/src/features/privacy-requests/dashboard/PrivacyRequestFiltersBar.tsx

```diff
@@ -0,0 +1,119 @@
+import dayjs from "dayjs";
+import {
+  AntDatePicker as DatePicker,
+  AntDisplayValueType as DisplayValueType,
+  AntFlex as Flex,
+  AntSelect as Select,
+} from "fidesui";
+
+import { DebouncedSearchInput } from "~/features/common/DebouncedSearchInput";
+import {
+  SubjectRequestActionTypeOptions,
+  SubjectRequestStatusOptions,
+} from "~/features/privacy-requests/constants";
+import { ActionType, PrivacyRequestStatus } from "~/types/api";
+
+interface PrivacyRequestFiltersBarProps {
+  filters: {
+    search: string | null;
+    from: string | null;
+    to: string | null;
+    status: PrivacyRequestStatus[] | null;
+    action_type: ActionType[] | null;
+  };
+  setFilters: (filters: {
+    search?: string | null;
+    from?: string | null;
+    to?: string | null;
+    status?: PrivacyRequestStatus[] | null;
+    action_type?: ActionType[] | null;
+  }) => void;
+}
+
+export const PrivacyRequestFiltersBar = ({
+  filters,
+  setFilters,
+}: PrivacyRequestFiltersBarProps) => {
+  // Convert filters to date range value
+  const getDateRange = (): [dayjs.Dayjs, dayjs.Dayjs] | null => {
+    if (filters.from && filters.to) {
+      return [dayjs(filters.from), dayjs(filters.to)];
+    }
+    return null;
+  };
+
+  const handleDateRangeChange = (
+    dates: [dayjs.Dayjs | null, dayjs.Dayjs | null] | null,
+  ) => {
+    const [from, to] = dates || [null, null];
+    setFilters({
+      from: from ? from.format("YYYY-MM-DD") : null,
```

Comment:
```suggestion
      from: from?.format("YYYY-MM-DD"),
```
ternary can be simplified